### ISSUE TITLE
Can't writePDB from trimmed ensemble in python3

### ISSUE THREAD

**Issue by jamesmkrieger** [prody/ProDy#800]
I think the ensemble was originally saved in python2 but not sure and that may or may not be related to the problem.

```
Python 3.7.1 (default, Dec 14 2018, 13:28:58) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.2.0 -- An enhanced Interactive Python. Type '?' for help.
Using matplotlib backend: Qt5Agg

In [1]: from prody import *                                                                                                                                  

In [2]: dali_ens = loadEnsemble('dali_PBP-I_filter_Z_above10_len_below50_percent_legacy_refined_1_10_rmsd_reordered_upgma_reversed.ens.npz')                 

In [3]: trimmed_dali_ens = trimPDBEnsemble(dali_ens, 0.7)                                                                                                    

In [4]: writePDB('trimmed_dali_PBP-I_ensemble.pdb', trimmed_dali_ens)                                                                                        
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-4-ab31a513366f> in <module>
----> 1 writePDB('trimmed_dali_PBP-I_ensemble.pdb', trimmed_dali_ens)

~/Documents/code/ProDy/prody/proteins/pdbfile.py in writePDB(filename, atoms, csets, autoext, **kwargs)
   1197         filename += '.pdb'
   1198     out = openFile(filename, 'wt')
-> 1199     writePDBStream(out, atoms, csets, **kwargs)
   1200     out.close()
   1201     return filename

~/Documents/code/ProDy/prody/proteins/pdbfile.py in writePDBStream(stream, atoms, csets, **kwargs)
   1067     for i, an in enumerate(atomnames):
   1068         if len(an) < 4:
-> 1069             atomnames[i] = ' ' + an
   1070 
   1071     s_or_u = np.array(['a']).dtype.char

TypeError: can only concatenate str (not "numpy.bytes_") to str
```

**Comment by jamesmkrieger** [comment]
We'll have a look and get back to you with a fix. Thanks for reporting this.

**Comment by jamesmkrieger** [comment]
This should be fixed by #848
